core/src/main/java/com/linecorp/armeria/internal/client/DnsUtil.java

```diff
@@ -98,4 +99,24 @@ public static void warnInvalidRecord(Logger logger, String logPrefix, DnsRecordT
     }
 
     private DnsUtil() {}
+
+    public static void encodeName(String name, ByteBuf out) {
```

Comment:
How about moving this method and the class to a test class because they are only used by tests?

Reply:
Ahh, never knew that this is used only by test classes. Sorry, @amitvc, but could we revert this?

Reply:
No problem @trustin. I reverted this particular change.

Reply:
Thanks, @amitvc 🙇